docs/connecting/ssh.md

```diff
@@ -37,35 +37,22 @@ If you know what to do with this information already, you can skip the rest of t
 !!! note
     Your username should be your email with all at's `@` and dot's `.` replaced with underscores `_`. View your username [here](https://unity.rc.umass.edu/panel/account.php)
```

Comment:
This is not strictly correct. Often times this is true, but we use EPPN (Edu-person principal name) to create usernames, and sometimes people will have emails that are not the same as their netids. (I know this isn't part of this PR but I just spotted it, so could you update that as well?)

```suggestion
    Your username is in the format `<organization username>_<organization>_edu`. View your username [here](https://unity.rc.umass.edu/panel/account.php)
```